qkeras/utils.py

```diff
@@ -630,7 +677,7 @@ def quantized_model_dump(model,
   for layer in model.layers:
     if layer.__class__.__name__ in [
         "QActivation", "Activation", "QDense", "QConv2D", "QDepthwiseConv2D",
```

Comment:
this list is reused in other places, could you try to refactor this as well.

Reply:
Done